.github/workflows/ci.yml

```diff
@@ -10,20 +10,43 @@ on:
   pull_request:  # for all PRs regardless of its base branch
 
 jobs:
-  build:
+  build-test:
 
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [10.x, 12.x, 14.x]
+    env:
+      CHAIN_ID: testing
+      MNEMONIC: ${{ secrets.TEST_MNEMONIC }}
```

Comment:
This loads a secret from the Github repo: https://docs.github.com/en/actions/reference/encrypted-secrets

Reply:
Oh, Can not looking `TEST_MNEMONIC`.
It's looks good to put confidential data in.